src/mpid/ch4/src/ch4_init.h

```diff
@@ -98,12 +98,72 @@ static inline int MPIDI_choose_netmod(void)
 #endif
 
 #undef FUNCNAME
+#define FUNCNAME MPIDI_Coll_selection_init
+#undef FCNAME
+#define FCNAME MPL_QUOTE(FUNCNAME)
+MPL_STATIC_INLINE_PREFIX int MPIDI_Coll_selection_init(void)
```

Comment:
nit: please use `MPIDI_[lowercase]` for function names. typedefs in the PR look good.